.github/workflows/ci-pytorch_test-conda.yml

```diff
@@ -38,6 +38,8 @@ jobs:
     - name: Workaround for https://github.com/actions/checkout/issues/760
       run: git config --global --add safe.directory /__w/lightning/lightning
 
+    - run: exit 0
```

Comment:
Quickly skimmed through the existing threads, but I don't think there's an easy way for a job to exit early. At this moment, I guess we need to skip all steps after this step. https://github.com/actions/runner/issues/662